cp3-networkpolicy/egress/license-service-reporter-manually/ibm-license-service-reporter-oidc-egress.yaml

```diff
@@ -10,6 +10,6 @@ spec:
   egress:
     - to:
         - ipBlock:
-            cidr: <CIDR of oidc-issuer-url> # Replace with actual CIDR range
+            cidr: ${OIDC-CIRD} # CIDR of OIDC
```

Comment:
```suggestion
            cidr: ${OIDC-CIDF}
```

Reply:
You have right. Variable have good enough name. Comment is unnecessary.